README.md

```diff
@@ -3,7 +3,8 @@
 # DiffKemp
 
 DiffKemp is a tool for automatic static analysis of semantic differences between
-different versions of the Linux kernel.
+different versions of projects written in C, with main focus on the Linux
```

Comment:
Now when this first paragraph mentions "different versions of projects", it might be better to say "There are two options to install the *tool*" in the following Installation section, to avoid confusion between DiffKemp and the compared project.

Another idea: I'd also consider calling DiffKemp *framework* here instead of *tool*, given the recent and future changes, like in Petr's NETSYS 2022 talk.